kmip/__init__.py

```diff
@@ -63,3 +64,11 @@
         logging.basicConfig()
 
 __all__ = ['core', 'demos', 'services']
+
+if sys.version_info[:2] == (2, 6):
+    warnings.simplefilter("always")
+    warnings.warn(
+        ("Please use a newer version of Python (2.7.9+ preferred). PyKMIP "
+         "support for Python 2.6 will be abandoned in the future."),
```

Comment:
s/abandoned/deprecated
*or*
...will no longer support Python 2.6 in the future